spec/docs/participant_declarations_spec.rb

```diff
@@ -101,6 +101,18 @@
       tags "participants declarations"
       security [bearerAuth: []]
 
+      parameter name: :filter,
+                in: :query,
+                schema: {
+                  "$ref": "#/components/schemas/ListFilter",
```

Comment:
That component is 
```
description: "Filter a list of records to return more specific results"
type: object
properties:
  updated_since:
    description: "Return only records that have been updated since this date and time (ISO 8601 date format)"
    type: :string
    example: "2021-05-13T11:21:55Z"
```
Which doesn't really match the filter in our case. Can we add participant_id field to it?